demos/select.html

```diff
@@ -69,13 +69,13 @@
           <span class="mdc-select__selected-text">Pick a food group</span>
           <div class="mdc-simple-menu mdc-select__menu">
             <ul class="mdc-list mdc-simple-menu__items">
-              <li class="mdc-list-item" role="option"  aria-disabled="true">
+              <li class="mdc-list-item" role="option" aria-disabled="true">
                 Pick a food group
               </li>
               <li class="mdc-list-item" role="option" tabindex="0">
                 Bread, Cereal, Rice, and Pasta
               </li>
-              <li class="mdc-list-item" role="option"  tabindex="0">
+              <li class="mdc-list-item" role="option" aria-disabled="true" tabindex="-1">
```

Comment:
I believe that disabled items should not be focusable. Also, `-1` for a tabindex doesn't allow users to tab to it; I believe it means that JS can `focus()` it if need be. So yeah I feel like removing it would be the better option.